app/api/helpers/scheduled_jobs.py

```diff
@@ -140,3 +140,12 @@ def send_event_fee_notification_followup():
                                                         app_name,
                                                         link,
                                                         incomplete_invoice.event.id)
+
+
+def expire_pending_tickets_after_one_day():
+    from app import current_app as app
+    with app.app_context():
+        db.session.query(Order).filter(Order.status == 'pending',
+                                       (Order.created_at - datetime.datetime.today()).days > 1).\
```

Comment:
`Order.created_at - datetime.datetime.today()` will always be negative

Reply:
@iamareebjamal changed it